pages.es/common/vimtutor.md

```diff
@@ -0,0 +1,11 @@
+# vimtutor
+
+> Vim tutor, enseña los comandos básicos de vim.
+
+- Ejecuta vim tutor utilizando el idioma especificado (en, fr, de, ...):
```

Comment:
```suggestion
- Ejecuta vim tutor utilizando el idioma especificado (en, es, de, ...):
```
I think it would make sense here to include `es` as available language.